engine/schema/src/main/resources/META-INF/db/schema-41700to41710.sql

```diff
@@ -20,4 +20,112 @@
 --;
 
 UPDATE `cloud`.`configuration` SET `value` = 'false' WHERE `name` = 'network.disable.rpfilter' AND `value` != 'true';
-UPDATE `cloud`.`configuration` SET `value` = 'false' WHERE `name` = 'consoleproxy.disable.rpfilter' AND `value` != 'true';
\ No newline at end of file
+UPDATE `cloud`.`configuration` SET `value` = 'false' WHERE `name` = 'consoleproxy.disable.rpfilter' AND `value` != 'true';
+
```

Comment:
why changes for the DB v 4.17.1 in this PR?

Reply:
Rebasing the branch and pushing brings changes from all authors including my own changes.
My changes are reserved for plugins/user-authenticators/oauth2.
I know this is not the right way to push my code, can you suggest how to do it the right way?
Maybe I'll create a new PR with only my changes and close this one.

Reply:
sorry @neogismm I do not understand the entire background. Are you saying you included code for other branches in this one? If you base your changes on main that shouldn´t happen. You can clean up your local working branch and force push it to your github branch to clean it up. There is no need for a new PR. 
But these schema changes for 4.17 are not wanted, i'd say.

Reply:
Sorry @DaanHoogland but I'm not able to figure out how to clean up this branch so I'm closing this PR and opening a new one that only includes my additions/changes to the OAuth feature.